packages/core/src/spec/lib/jdk/JdkHelperSpec.ts

```diff
@@ -63,4 +64,86 @@ describe('JdkHelper', () => {
       expect(env['JAVA_HOME']).toBe('C:\\Users\\user\\jdk8\\');
     });
   });
+
+  describe('validatePath', () => {
+    it('Creates a Windows environment and checks that a valid path will pass', async () => {
+      mock({
+        'jdk': {
+          'release': 'JAVA_VERSION="1.8',
+        }});
+      expect((await JdkHelper.validatePath('jdk')).isOk()).toBeTrue();
+      mock.restore();
+    });
+
+    it('Creates a Windows environment and checks that a valid path will not pass', async () => {
```

Comment:
"checks that a valid path will not pass" ->
"checks that an invalid path will not pass"